app/src/organisms/DropTipWizard/BeforeBeginning.tsx

```diff
@@ -16,47 +16,113 @@ import {
   DISPLAY_INLINE_BLOCK,
   JUSTIFY_SPACE_BETWEEN,
   PrimaryButton,
+  JUSTIFY_FLEX_END,
+  POSITION_ABSOLUTE,
 } from '@opentrons/components'
 import { StyledText } from '../../atoms/text'
-import { NeedHelpLink } from '../CalibrationPanels'
+import { SmallButton, MediumButton } from '../../atoms/buttons'
+// import { NeedHelpLink } from '../CalibrationPanels'
+
+import blowoutVideo from '../../assets/videos/droptip-wizard/Blowout-Liquid.webm'
+import droptipVideo from '../../assets/videos/droptip-wizard/Drop-tip.webm'
 
 // TODO: get help link article URL
-const NEED_HELP_URL = ''
+// const NEED_HELP_URL = ''
 
 interface BeforeBeginningProps {
   handleCreateAndSetup: (shouldDispenseLiquid: boolean) => void
   isCreateLoading: boolean
+  isOnDevice: boolean
 }
 
 export const BeforeBeginning = (
   props: BeforeBeginningProps
 ): JSX.Element | null => {
-  const { handleCreateAndSetup, isCreateLoading } = props
+  const { handleCreateAndSetup, isCreateLoading, isOnDevice } = props
   const { i18n, t } = useTranslation(['drop_tip_wizard', 'shared'])
   const [flowType, setFlowType] = React.useState<
-    'liquid_and_tips' | 'only_tips'
-  >('liquid_and_tips')
+    'liquid_and_tips' | 'only_tips' | null
+  >(null)
   const handleProceed = (): void => {
     handleCreateAndSetup(flowType === 'liquid_and_tips')
   }
 
-  return (
+  return isOnDevice ? (
```

Comment:
```suggestion
  if (isOnDevice) return (
```
Nit, but I'd probably break this into separate statements here. A ternary does work, but it's a bit harder to read when dealing with components this large. This goes for every other step component, too.